releasing/localbuild.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
Rats, I'm so sorry... ironically, on Windows it's easy to miss this. I did this first time around too but subsequent rewrites ended up losing it in the index (since Win has no such concept at the file level). 🤦‍♂️

Reply:
Also just FYI, turns out `cloudbuild-local.sh` and needed to be made executable. I noticed `helpers.sh` isn't but that's outside scope of this issue, but just FYI. The `README.md` tells folks to `source` it but I could see people wanting to use `./helpers.sh` as well.

Reply:
Updated, summarizing below.